test/fixtures/vcr/relay_group_add_relay.json

```diff
@@ -11,14 +11,14 @@
       "url": "http://localhost:4000/v1/token"
     },
     "response": {
-      "body": "{\"token\":{\"value\":\"w05n5iZL4peQoZ1gibP1MR1TgvSTXrIwZzCIJYRjR+0=\"}}",
```

Comment:
@christophermaier Oh, I'm not arguing that unnecessary changes shouldn't be dropped. When I wrote the tests vcr complained and would not run until I regenerated the cassettes. I'm just saying that I don't know how to prevent that from happening.